spec/vehicle_spec.rb

```diff
@@ -33,4 +71,44 @@
       expect(@camaro.electric_vehicle?).to eq(false)
     end
   end
+
+  describe '#add_registration' do
+
+    it 'Can add a registration date' do
+
+      expect(@cruz.registration_date).to eq(nil)
+      expect(@bolt.registration_date).to eq(nil)
+      expect(@camaro.registration_date).to eq(nil)
+
+      @cruz.add_registration
+      expect(@cruz.registration_date).to be_a(Integer)
+      expect(@bolt.registration_date).to eq(nil)
+      expect(@camaro.registration_date).to eq(nil)
+      
+      @bolt.add_registration
+      expect(@cruz.registration_date).to be_a(Integer)
+      expect(@bolt.registration_date).to be_a(Integer)
+      expect(@camaro.registration_date).to eq(nil)
+      
+    end
+  end
+
+  describe '#add_plate' do
+    it 'can add a plate type' do
+
+      expect(@cruz.plate_type).to eq(nil)
+      expect(@bolt.plate_type).to eq(nil)
+      expect(@camaro.plate_type).to eq(nil)
+      
+      @cruz.add_plate(:regular)
```

Comment:
Nice job testing your `add_plate` method here. This would be considered a unit test.